hive_metastore_client/hive_metastore_client.py

```diff
@@ -131,7 +131,11 @@ def add_partitions_to_table(
             table_partition_keys=table.partitionKeys,
         )
 
-        self.add_partitions(partition_list_with_correct_location)
+        try:
+            self.add_partitions(partition_list_with_correct_location)
+            return True
+        except AlreadyExistsException:
```

Comment:
what if we added a log message here? for later debugging purposes

Reply:
Duplicate. Closing it.